src/Sylius/Bundle/FlowBundle/Process/Builder/ProcessBuilder.php

```diff
@@ -79,6 +79,10 @@ public function add($name, $step)
             throw new \InvalidArgumentException('Step added via builder must implement "Sylius\Bundle\FlowBundle\Process\Step\StepInterface"');
         }
 
+        if (false === $step->isActive()) {
```

Comment:
Well, for me as ``isActive()`` **must** return ``bool``, there is no need for strict check. If it returns something else (even if it can be casted to ``bool``, as empty string or ``0``) it is a problem of bad code implementation, however not here, but in ``isActive()`` method. But it's only my opinion, I'll be happy to read others minds ;)